packages/node_modules/@webex/plugin-meetings/test/utils/testUtils.js

```diff
@@ -134,11 +134,13 @@ const waitForEvents = (scopeEvents, timeout = max) => {
               }
             }
 
-            if (obj.match && obj.match(value)) {
-              clearTimeout(timer);
-              obj.scope.off(obj.event, handler);
+            if (obj.match) {
```

Comment:
this change is needed, because when `match` callback is defined we don't want to resolve when the number of received events equals `scopeEvents.length` - we only want to resolve once the `match()` callback succeeds

Reply:
&& operator makes that happen tho !?

Reply:
no, the problem is that if both conditions are together wrapped with a single "if" then in the case where `obj.match` is defined but `obj.match(value)` returns false we go to the "else if": `else if (result.length === scopeEvents.length) {` and we don't want that to happen in this case.